### PR TITLE
feat: add git commands #80

### PR DESCRIPTION
**Pull request by janetthieu**
### Checklist:
- [x] Read our [contributing guidelines](../docs/CONTRIBUTING.md).
- [x] Search for duplicates.
- [x] Used an informative name for this pull request.

### Follow-up
- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: CheatSheets/git-cheatsheet.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ created: 2022-10-18
 | `gitk` | Shows graphical interface for a local repository |
 | `git revert [commit-ID]` | Create new commit, reverting the changes from a specified commit |
 | `git checkout [file]` | Matches the file with last commit |
+| `git help` | Shows list of the most commonly used Git commands |
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:69]:
```suggestion
| `git help -a` | Shows the list of the most commonly used Git commands |
```

File: CheatSheets/git-cheatsheet.md [hunk 2]
```diff
@@ -66,6 +66,8 @@ created: 2022-10-18
...
+| `git restore` | Restore working tree files |
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:70]:
```suggestion
| `git restore .` | - Restore working tree files <br /> - To restore all files in the current directory |
```
| Command | Description |
| ------------ | ------------ |
| `git restore .` | - To restore all files in the current directory <br /> - Restore working tree files|

File: CheatSheets/git-cheatsheet.md [hunk 3]
```diff
@@ -124,6 +126,8 @@ created: 2022-10-18
 | `git bisect start` | Starts the bisection search process to find that bad commit which introduced the bug we're facing right now |
 | `git bisect good` <br /> `git bisect good [Commit ID]` | Takes up the good commit, which is that one where the bug was not there |
 | `git bisect bad [Commit ID]` | Takes up the bad commit, which is that one where the bug was there. If commit ID is not provided, then it takes up the current commit as the bad commit |
+| `git grep` | Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in given tree objects |
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:129]:
```suggestion
| `git grep "hello"` |  - A text search on all files in the directory <br /> - Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in the given tree objects |
```
| Command | Description |
| ------------ | --------------- |
| `git grep "hello"` |  - A text search on all files in the directory <br /> - Print lines matching a specified pattern in the tracked files in the work tree, blobs registered in the index file, or blobs in the given tree objects |

File: CheatSheets/git-cheatsheet.md [hunk 4]
```diff
@@ -124,6 +126,8 @@ created: 2022-10-18
...
+| `git bugreport` | Collect information (e.g., user's machine, Git client, and repository state) for user to file a bug report |
```
Comment (crescentpartha) [inline CheatSheets/git-cheatsheet.md:130]:
```suggestion
| `git bugreport` | - Created new report at `git-bugreport-2022-10-25-1228.txt` <br /> - Collect information (e.g., user's machine, Git client, and repository state) for user to file a bug report |
```
| Command | Description |
| --------- | -------------- |
| `git bugreport` | - Created new report at `git-bugreport-2022-10-25-1228.txt` <br /> - Collect information (e.g., user's machine, Git client, and repository state) for user to file a bug report |

**Review by crescentpartha — CHANGES_REQUESTED** [review]
Please, modify your code based on the below comments. @janetthieu

**Comment by janetthieu** [comment]
I made the modifications per your suggestions. Please let me know if there is anything else. Thank you, @crescentpartha!

**Review by crescentpartha — APPROVED** [review]
Approved your changes.

**Pull request merged**